softwarereview_intro.pt.Rmd

```diff
@@ -1,63 +1,63 @@
 
-Estamos cientes de que os sistemas abertos podem ter desvantagens. Por exemplo, na revisão acadêmica tradicional, [a revisão por pares duplo-cega pode aumentar a representação de autores do sexo feminino](https://www.sciencedirect.com/science/article/pii/S0169534707002704) sugerindo preconceito em revisões não cegas. Também é possível que os revisores sejam menos críticos na revisão aberta. No entanto, afirmamos que a abertura da conversa sobre a revisão proporciona uma verificação da qualidade e da parcialidade da revisão; é mais difícil injetar comentários subjetivos ou sem embasamento em público e sem a cobertura do anonimato. Em última análise, acreditamos que a comunicação direta e pública entre autores e revisores melhora a qualidade e a imparcialidade das revisões.
+Estamos cientes de que sistemas abertos podem ter desvantagens. Por exemplo, na revisão acadêmica tradicional, [a revisão por pares duplo-cega pode aumentar a representação de pessoas autoras do gênero feminino](https://www.sciencedirect.com/science/article/pii/S0169534707002704), sugerindo viés em revisões não-cegas. Também é possível que revisores(as) sejam menos críticos em revisões abertas. No entanto, sustentamos que a abertura da conversa de revisão funciona como um controle de qualidade e viés; é mais difícil inserir comentários subjetivos ou sem embasamento em público e sem o manto do anonimato. Em última análise, acreditamos que a comunicação direta e pública entre autores(as) e revisores(as) melhora a qualidade e a equidade das avaliações.
 
-Além disso, os autores e revisores podem entrar em contato com os editores em particular se tiverem alguma dúvida ou pergunta.
+Além disso, os(as) autores(as) e revisores(as) podem entrar em contato com os(as) editores(as) em particular se tiverem alguma dúvida ou questão.
 
 ## Como os usuários saberão que um pacote foi revisado? {#how-will-users-know-a-package-has-been-reviewed}
 
-- O LEIAME do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
-- Seu pacote receberá um selo [`docs.ropensci.org` site de documentos](#rodocsci) que você pode vincular a partir da DESCRIÇÃO.
-- O repositório do seu pacote será transferido para a organização rOpenSci.
-- Se os revisores [concordarem em ser listados na DESCRIÇÃO](#authorship) seus metadados mencionarão a revisão.
+- O README do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
+- Seu pacote terá um [site de documentação em `docs.ropensci.org`](#rodocsci) que você pode referenciar no DESCRIPTION.  
+- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.
+- Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), seus metadados mencionarão a revisão.
```

Comment:
The phrase is not 100% clear to me.
In "their metadata will mention the review.", "their metadata" is referring to the package metadata? 
If so, we can make if more explicit:
```suggestion
- Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), os metadados do pacote mencionarão a revisão.
```